cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,38 @@
+        data_source = request.args.get("data_source")
+
+        db = firestore.Client()
+        stream_data = (
+            db.collection("users")
+            .document(uid)
+            .get()
+            .to_dict()[f"stream={data_source}"]
+        )
+
+        if not refresh_token:
+            raise ValueError(f"No refresh token found for user {uid}")
```

Ruff (0.8.2)

Abstract `raise` to an inner function

(TRY301)